src/routes/lostItems/lostItem.route.js

```diff
@@ -1,14 +1,15 @@
 import express from 'express';
 import authentication from '../../middlewares/authentication';
 import lostItemsController from '../../controllers/lostDoc';
-import docValidation from '../../middlewares/docValidation';
+import { docValidation, editValidation} from '../../middlewares/docValidation';
 
 const lostRouter = express.Router();
 
 lostRouter
   .post('/lost', authentication, docValidation, lostItemsController.lostItem)
   .post('/found', authentication, docValidation, lostItemsController.foundItem)
   .get('/lost', authentication, lostItemsController.allLost)
-  .get('/found', authentication, lostItemsController.allFound);
+  .get('/found', authentication, lostItemsController.allFound)
+  .put('/edit/:_id', authentication, editValidation, lostItemsController.updateItem);
```

Comment:
Hey @SimonImanigirimpuhwe we don't need another route for edit. It is obvious that when you send `put` request to a particular route you are intended to edit a resource, according to Restful API, each method has its specific intended action. You can read more on this here [RESTful API](https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/)

Reply:
It means that we don't need another route, for `put` just do `.put(/lost/:_id)` and we are good to go.